app/scripts/card.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+import $ from 'jquery';
+
+$('.js-card-queue-toggle').click(function (e) {
+
+  e.preventDefault();
+
+  $(this).closest('.js-card-queue').toggleClass('mdl-card--expanded');
```

Comment:
In the future you can use this: https://github.com/viniciusalmeida/jQuery.toggleModifier 😉